jujugui/static/gui/src/app/app.js

```diff
@@ -1187,6 +1196,11 @@ YUI.add('juju-gui', function(Y) {
           showPay={window.juju_config.payFlag || false}
           updateCloudCredential={
             controllerAPI.updateCloudCredential.bind(controllerAPI)}
+          updateAddress={
```

Comment:
We have this conversation every month or so and we keep coming back to passing the methods down. @hatched thoughts?